### DISCUSSION TITLE
User identification when connecting

### DISCUSSION THREAD

**Discussion by mrgreiner** [dbeaver/dbeaver#34338] [Q&A] [answered]
Hi,

I'm setting connections to my servers, but have a problem. When I connect to my server (in this case MariaDB in Linux), it is using "name"@"station name" as identification. I would like dbeaver to use "name"@"ip address" instead. I couldn't find where I set this. Can someone help me on this?

Thank you,

Roberto

**Comment by E1izabeth** [comment, marked as answer]
You can set it in `Connection settings->Client identification`

![image](https://github.com/dbeaver/dbeaver/assets/28875055/3163fccf-96c5-46a5-a915-13bcec800b68)
